components/ipfs/ipfs_utils.cc

```diff
@@ -139,6 +139,11 @@ bool HasIPFSPath(const GURL& gurl) {
                              (path.find("/ipns/") != std::string::npos));
 }
 
+bool HasIPNSPath(const GURL& gurl) {
+  const auto& path = gurl.path();
+  return gurl.is_valid() && ((path.find("/ipns/") != std::string::npos));
```

Comment:
maybe `return gurl.is_valid() && base::Contains(gurl.path(), "/ipns/");`

also it seems `HasIPFSPath` above includes `HasIPNSPath`, is that intended?